tutor-contrib-multi-plugin/tutor_multi_k8s_plugin/plugin.py

```diff
@@ -28,16 +29,21 @@
         # We are using HTTPS
         "ENABLE_HTTPS": True,
     },
+    "unique": {
+        "ELASTICSEARCH_HTTP_AUTH": "{{K8S_NAMESPACE}}:{{ 24|random_string }}",
+        "ELASTICSEARCH_INDEX_PREFIX": "{{K8S_NAMESPACE}}-{{ 4|random_string }}",
```

Comment:
While I was testing the string generated included uppercase characters so when I tried re-indexing a course I got an 'invalid index name must be lowercase' error. Maybe we should normalize this string?

Reply:
Ah, I got lucky that it generated a lowercase string for me. I've updated the filter.